Tools/Scripts/webkitpy/benchmark_runner/http_server_driver/http_server/signposts/signposts.c

```diff
@@ -0,0 +1,69 @@
+#define PY_SSIZE_T_CLEAN
+#include <Python.h>
+
+#include <os/log.h>
+#include <os/signpost.h>
+#include <stdio.h>
+
+#define SIGNPOST_NAME "JSGlobalObject"
```

Comment:
This is "JSCJSGlobalObject" for Safari, so I think I should switch it in case tools rely on the exact string